packages/kbn-unified-data-table/src/components/source_document.tsx

```diff
@@ -60,6 +63,7 @@ export function SourceDocument({
       compressed
       className={classnames('unifiedDataTable__descriptionList', CELL_CLASS)}
       data-test-subj={dataTestSubj}
+      css={customStyles}
```

Comment:
**nit**: why not keep the same naming for this property as it's widely used across the codebase? Renaming to `customStyles` makes me think it will set the `style` property from a consumer perspective and can be confusing when TS then tells you it expects a `css` type property.

Reply:
You mean to set it as `customCss` ? or simply `css`